src/components/composer/composer/composer.ts

```diff
@@ -400,6 +400,35 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
     this.processTokenAtCursor();
   }
 
+  onDblClick() {
+    if (this.env.model.getters.isReadonly()) {
+      return;
+    }
+    const composerContent = this.env.model.getters.getCurrentContent();
+    const isValidFormula =
+      composerContent.startsWith("=") && this.env.model.getters.getCurrentTokens().length > 0;
+
+    if (isValidFormula) {
+      const tokens = this.env.model.getters.getCurrentTokens();
+      const currentSelection = this.contentHelper.getCurrentSelection();
+      if (currentSelection.start === currentSelection.end) return;
+
+      const currentSelectedText = composerContent.substring(
+        currentSelection.start,
+        currentSelection.end
+      );
+      const token = tokens.filter((line) => line.value.includes(currentSelectedText))[0];
```

Comment:
Looking for the first token which includes the selected text doesn't work properly when several tokens contains the same text.
E.g. `=A1+A1` and double click the second "A1" will actually select the first one.
You can probably play around with `token.start` and `token.end`